bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -632,6 +632,15 @@ impl Client {
         })
     }
 
+    /// Deletes a pusher of given pusher ids
+    pub fn delete_pusher(&self, identifiers: PusherIdentifiers) -> Result<(), ClientError> {
+        RUNTIME.block_on(async move {
+            let ids = identifiers.into();
+            self.inner.delete_pusher(ids).await?;
+            Ok(())
+        })
+    }
```

Comment:
We now support async functions via FFI, so it's better to use this pattern:

```suggestion
    pub async fn delete_pusher(&self, identifiers: PusherIdentifiers) -> Result<(), ClientError> {
        self.inner.delete_pusher(identiers.into()).await?;
        Ok(())
    }
```

If you feel brave enough, you can also update `set_pusher` to be async :-).